samples/BoltOn.Samples.Infrastructure/BoltOn.Samples.Infrastructure.csproj

```diff
@@ -16,9 +16,9 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.2.0" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="2.2.0" />
     <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="2.5.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />

```

Comment:
@gokulm : what about using  System.Text.Json builtin for .net core ? looks like thats more efficient i guess it would be available from .net core 3.0!

Reply:
@skankanvar good to know that!

Reply:
on less depedency.